ruby_on_rails/app_initialisation.md

```diff
@@ -1,61 +1,77 @@
 # Initialise the Rails App
 
-* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v` and update it if you are not.
-Run `rbenv versions` to check your installed ruby versions.
+## Default Rails setup
+
+* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v`
+and update it if you are not. Run `rbenv versions` to check your installed ruby versions.
 Follow the instructions on screen to eventually install it.
 
+* Switch the global Ruby version to the newest one with `rbenv global <ruby-version>` so that the latter commands
+run in the correct context.
+
 * Run `gem install bundler` to install Bundler.
 You may have it already installed, but this command will make sure that you have the latest version.
 
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
-You can do this with `gem update rails`.
+You can do this with `gem update rails`. Beware of beta versions.
 
-* Start a new Rails project using `rails new [project-name] -d postgresql --skip-sprockets --webpack` where the `project-name` is exactly the one you chose before.
+* Start a new Rails project using `rails new [project-name] --database=postgresql --skip-test --skip-sprockets --webpacker` where the `project-name` is exactly the one you chose before.
 You may want to choose a different database from Postgres, but most of the time that will be your choice.
 If you do not need a DB you may rethink the fact that you may not need Rails at all :) Take a look at [Sinatra](http://www.sinatrarb.com/) or [Angular](https://angular.io/)
 
-* Specify the ruby version in the project's `Gemfile` with
+* Update the Ruby version in `.ruby-version` and load it in the fresh project's `Gemfile` like this
```

Comment:
the ruby version should already be the correct one, why do I need to update it?

Reply:
Correct. This was before I added https://github.com/renuo/applications-setup-guide/pull/174/files#diff-fd44d3d24bdd2374889b16ee4e31adfeR9